krun/util.py

```diff
@@ -528,3 +528,35 @@ def check_audit_unchanged(results, platform):
             "identical to the one on which the last results were "
             "gathered, which is not the case.")
         fatal(error_msg)
+
+
+def daemonise():
+    """Daemonise Krun"""
+
+    debug("daemonising...")
+    try:
+        pid = os.fork()
+    except OSError:
+        fatal("failed to daemonise: first fork")
+
+    if pid != 0:
+        # parent
+        os._exit(0)
+
+    # child becomes the session leader
```

Comment:
Not exactly true -- the child *creates* a new session (according to the man page). I think it's probably best to remove this comment (`man setsid` is more informative).

Reply:
Eh?

OpenBSD manual:
> The calling process is the session leader of the new session

Linux manual:
> The calling process is the leader  of  the  new  session

right?

Reply:
Yes, but the current comment can be read to suggest it's taking over an *existing* session rather than a *new* session. But ultimately the comment isn't really useful. Anyone who needs to know this should look at the man page.